File: benchmarks/cross/progs/jsonb.py
import json
veri = [{"id": i, "ad": "oge", "puan": i * 2, "aktif": True} for i in range(500)]
toplam = 0
for _ in range(100):
    s = json.dumps(veri)
    geri = json.loads(s)
    toplam += len(geri)
print(toplam)

File: benchmarks/cross/progs/jsonb.js
const veri = [];
for (let i = 0; i < 500; i++) veri.push({id: i, ad: "oge", puan: i * 2, aktif: true});
let toplam = 0;
for (let k = 0; k < 100; k++) {
    const s = JSON.stringify(veri);
    const geri = JSON.parse(s);
    toplam += geri.length;
}
console.log(toplam);
